src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -2,7 +2,11 @@ import React from "react";
 
 import "./buttonGroup.css";
 
-const ButtonGroup = props => {
+type Props = {
+  className?: string;
```

Comment:
I think if `className` is the only prop, you can use the type `HTMLProps<HTMLDivElement>` directly in your `FunctionComponent` declaration